src/styles/blocks/_card.scss

```
@@ -0,0 +1,42 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: block;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $background-button-color;
+  height: 406px;
+}
+
+.image-imac {
+  height: 134px;
+  width: 160px;
+  padding: 32px 19px 36px;
+}
+
+.text_information {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 0 16px;
```

to make outer indent you should use margin (not padding). Since we reset default margin and add padding for card you need just to use `margin-bottom` for card elements
```suggestion
  margin-bottom: 24px; //this is an example. Set a correct value from figma. 
```